bin/host/src/cli/tracing_util.rs

```diff
@@ -6,17 +6,15 @@ use tracing::Level;
 /// Initializes the tracing subscriber
 ///
 /// # Arguments
-/// * `verbosity_level` - The verbosity level (0-4)
+/// * `verbosity_level` - The verbosity level (0-2)
 ///
 /// # Returns
 /// * `Result<()>` - Ok if successful, Err otherwise.
 pub fn init_tracing_subscriber(verbosity_level: u8) -> Result<()> {
     let subscriber = tracing_subscriber::fmt()
         .with_max_level(match verbosity_level {
-            0 => Level::ERROR,
-            1 => Level::WARN,
-            2 => Level::INFO,
-            3 => Level::DEBUG,
+            0 => Level::INFO,
+            1 => Level::DEBUG,
             _ => Level::TRACE,
```

Comment:
Why? This will require showing both warnings and info by default now. Is this intentional?

Reply:
Ya totally fine by me. Downstream users can roll their own CLI if they want to configure tracing differently.